functional_tests/steps/home_page.py

```diff
@@ -0,0 +1,17 @@
+from behave import then, when  # pylint: disable=E0611
+from behave.runner import Context
+from playwright.sync_api import expect
+
+
+@when("An external user navigates to the ONS beta site homepage")  # pylint: disable=E1102
+def external_user_navigates_to_beta_homepage(context: Context) -> None:
+    context.page.goto(context.base_url)
+
+
+@then("they can see the beta homepage")  # pylint: disable=E1102
```

Comment:
```suggestion
@then("they can see the beta homepage")  # pylint: disable=not-callable
```

imho this is easier to grok and doesn't require one to look for the codes